### ISSUE TITLE
error: failed to fetch `https://github.com/rust-lang/crates.io-index` when emerging dev-util/cargo-0.17.0::rust

### ISSUE THREAD

**Issue by Alessandro-Barbieri** [gentoo/gentoo-rust#247]
```
>>>  +sandbox(compile)
>>> Compiling source in /var/tmp/portage/dev-util/cargo-0.17.0/work/cargo-0.17.0 ...
make -j5 -l5 VERBOSE=1 PKG_CONFIG_PATH=
/usr/bin/rustc -V
rustc 1.16.0
/var/tmp/portage/dev-util/cargo-0.17.0/work/cargo-0.17.0/target/snapshot/bin/cargo --version
cargo 0.13.0-nightly (e713e7f 2016-08-31)
/var/tmp/portage/dev-util/cargo-0.17.0/work/cargo-0.17.0/target/snapshot/bin/cargo build --target x86_64-unknown-linux-gnu --manifest-path /var/tmp/portage/dev-util/car
go-0.17.0/work/cargo-0.17.0//Cargo.toml --release  --verbose
    Updating registry `https://github.com/rust-lang/crates.io-index`
warning: spurious network error (2 tries remaining): [12/-1] Failed to resolve address for github.com: Temporary failure in name resolution
warning: spurious network error (1 tries remaining): [12/-1] Failed to resolve address for github.com: Temporary failure in name resolution
error: failed to fetch `https://github.com/rust-lang/crates.io-index`

Caused by:
  [12/-1] Failed to resolve address for github.com: Temporary failure in name resolution
make: *** [Makefile:130: cargo-x86_64-unknown-linux-gnu] Error 101
```

**Comment by mpkh** [comment]
@Alessandro-Barbieri weak... try same version from tree, not from overlay, they are very close...

**Comment by Alessandro-Barbieri** [comment]
ok, the version from tree works

**Comment by mpkh** [comment]
obsolete
